.github/workflows/docker-image-perception-develop.yml

```diff
@@ -88,32 +88,74 @@ jobs:
         run: |
           docker push --all-tags ${GITHUB_REMOTE_URL}/${IMAGE_NAME}
 
-  notify-success:
-    name: "Notify Slack - Success"
+  build-ros-perception-cuda-x86-64:
+    name: "Build ROS perception CUDA X86/64"
     runs-on: ubuntu-latest
-    needs: [build-ros-perception-x86-64, build-ros-perception-arm64]
-    if: success()
+    env:
+      BASE_IMAGE: "docker.pkg.github.com/sonia-auv/sonia_common/sonia_common_cuda:x86-perception-latest"
+      ARCH: x86
+      TARGET_TYPE: perception
+      TARGET_VERSION: develop
+      IMAGE_NAME: proc_image_processing_cuda
+      GITHUB_REMOTE_URL: docker.pkg.github.com/${{ github.repository }}
```

Comment:
remove space inside ${{ github.repository }}

Reply:
I'm following GitHub's syntax: https://docs.github.com/en/actions/reference/environment-variables

Reply:
It's also a copy-paste from what SONIA did